### PR TITLE
Add the libfuse package

### PR DESCRIPTION
**Pull request by asafkahlon**
Signed-off-by: Asaf Kahlon <asafka7@gmail.com>

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/libfuse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - meson
+    - pkg-config
```
Comment (scopatz) [inline recipes/libfuse/meta.yaml]: You should also have a ninja dep here

File: recipes/libfuse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libfuse3${SHLIB_EXT}
+    - test -f ${PREFIX}/include/fuse3/fuse.h
+
+about:
+  home: https://github.com/libfuse/libfuse
+  license: LGPL-2.1
+  license_file: LICENSE
```
Comment (scopatz) [inline recipes/libfuse/meta.yaml]: Please add a `license_family` entry

**Comment by asafkahlon** [comment]
@scopatz Thanks for the quick review :)
Pushed another revision.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**